cmake/DARTFindBullet.cmake

```diff
@@ -12,6 +12,7 @@
 find_package(Bullet COMPONENTS BulletMath BulletCollision MODULE QUIET)
 
 if((BULLET_FOUND OR Bullet_FOUND) AND NOT TARGET Bullet)
+  cmake_policy(SET CMP0057 NEW)
```

Comment:
The error could occur anywhere `IN_LIST` is used. Also, this policy setting isn't necessary if the required CMake version is set from the parent project (e.g., gazebo). Let's fix this in `DARTConfig.cmake.in` like [this one](https://github.com/dartsim/dart/pull/1434/files#diff-90586bbb8c55cc07024442a0d1fa71feR13-R26).

Reply:
ok, I'll copy that approach here

Reply:
fc749f6

Reply:
Awesome, thanks!